community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/staging/Step.java

```diff
@@ -93,4 +93,6 @@
      */
     @Override
     void close() throws Exception;
+
+    long doneBatches();
```

Comment:
This method is not called anywhere. Can we get rid of it?
And if I'm wrong, can we haz javadoc?